lib/tasks/represent_downstream.rake

```diff
@@ -55,13 +55,14 @@ namespace :represent_downstream do
   end
 
   desc "
-  Represent an individual edition downstream
+  Represent an individual or multiple editions downstream
   Usage
   rake 'represent_downstream:content_id[57a1253c-68d3-4a93-bb47-b67b9b4f6b9a]'
   "
   task :content_id, [:content_id] => :environment do |_t, args|
+    content_ids = args[:content_id].split(" ")
```

Comment:
Yeah - I based it on specialist publishers similar commands: https://github.com/alphagov/specialist-publisher/blob/master/lib/tasks/republish.rake and did notice (rather lately) that there had been confusion over their usage [here](https://deploy.publishing.service.gov.uk/job/run-rake-task/1155/console) (which may mean I tried to fix way more content_ids than needed)

The problem I have with comma is that it's a small step to ", " and then more whitespace concerns. Do we have anything in Publishing API tasks that sets a consistency here?